requirements.txt

```diff
@@ -4,3 +4,4 @@ scikit-learn==0.23.2
 python-dateutil==2.8.0
 pandas==1.2.4
 tsfresh==0.18.0
+statsmodels==0.12.2
```

Comment:
Good catch. Might be good to use `statsmodels==0.9.0` for consistency with the container library (https://github.com/aws/sagemaker-scikit-learn-container/blob/master/docker/0.23-1/extension/Dockerfile.cpu#L10)

Reply:
Update: I changed to statsmodel 0.9.0, but then some tests started failing. It looks like statsmodel 0.9 is using an older version of pandas with deprecated functions. To make that work I'd need to add yet another dependency of an older version of pandas (0.9 when pandas has versions > 1). I'd rather not do that if I don't have to.